config/tech-docs.yml

```diff
@@ -9,7 +9,7 @@ phase:
 
 # Links to show on right-hand-side of header
 header_links:
-  Control panel: https://controlpanel.services.alpha.mojanalytics.xyz/
+  Control panel: https://controlpanel.services.analytical-platform.service.justice.gov.uk/
```

Comment:
we're leaving the control panel link as the old one until everyone is migrated so please revert this